### PR TITLE
Feature/113 static input fields

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [EQ-113](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-113)

## Main Changes:
* Added inputs for all relevant domain values.
* Added client-side aliases to the `domainValues` context object for fields that share domain values with other fields.
* Restructured the state management to reduce boilerplate.
* Added a mapping file for the filter fields to facilitate their rendering.
* Removed `locationText` from the domain values. This domain has a large number of values, and filtering them will require querying the database.
* Added logic for removing duplicate values from the set retrieved from the domain service.

## Steps To Test:
1. Run the domain values ETL task.
2. Navigate to the app's home page: [http://localhost:3000](http://localhost:3000)
3. Confirm that the `assessments` and `actions` profile are included in the **Data Profile** drop-down.
4. Confirm that filter fields change when selecting profiles, and that the displayed fields are members of the selected profile.
5. Check the spacing of the grid layout in the **Filters** section. This will be rearranged in the future, as will spacing between sections, but the current items should flow properly.

### CODE REVIEW

File: app/client/src/routes/home.tsx [hunk 1]
```diff
@@ -100,61 +209,76 @@ function getInputValue(input: Exclude<InputState[keyof InputState], null>) {
   return input;
 }
 
+function getLocalStorageItem(item: string) {
+  if (storageAvailable()) {
+    return localStorage.getItem(item) ?? null;
+  } else return null;
+}
```
Comment (cschwinderg) [inline app/client/src/routes/home.tsx:216]:
```suggestion
function getLocalStorageItem(item: string) {
  return localStorage.getItem(item) ?? null;
}
```

The call to `storageAvailable` is causing the `__storage_test__` item to be written and removed from local storage on every render. I think we should just remove the call to `storageAvailable` as we shouldn't need to check wether storage is available to get an item out of local storage.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks great so far! 

I think the options in the select boxes should be alphabetized. This should probably be done in the ETL side.

File: app/client/src/routes/home.tsx (hunk 1 above)
Comment (maxdiebold-erg) [reply app/client/src/routes/home.tsx:216]: Good point, I'll change that.

**Comment by cschwinderg** [comment]
@maxdiebold-erg Can you look into the codesmells? It looks like sonarcloud was updated recently and is now more strict.

**Comment by cschwinderg** [comment]
We will want to hold off on merging this, because the ETL process is currently running. The ETL process will probably take about 1.5 hours to complete.

**Review by cschwinderg — APPROVED** [review]
Thanks for making those updates!

Attains is not stable at the moment, so I canceled the ETL. Feel free to merge this in, whenever you are ready.

**Pull request merged**